### DISCUSSION TITLE
Step-back / Time Travel Debugging

### DISCUSSION THREAD

**Discussion by karthiknadig** [microsoft/debugpy#1200] [Enhancements]
Support for the step back command.

**Comment by fabioz** [comment]
Is this the same as the `go to` (which is already done)?

**Comment by int19h** [comment]
This is a very high-level item for something like this:
https://devblogs.microsoft.com/visualstudio/introducing-time-travel-debugging-for-visual-studio-enterprise-2019/

**Comment by fabioz** [comment]
Thanks for the clarification.

**Comment by MikeBishop** [comment]
I have a program where the paired device will abandon the connection if the program is unresponsive for more than a few seconds, but I can only repro the problem with the paired device connected and active.  I would *really* like to be able to be able to capture and step through a run without halting the program execution.
